src/view/com/auth/SplashScreen.tsx

```diff
@@ -17,9 +28,32 @@ export const SplashScreen = ({
 }) => {
   const pal = usePalette('default')
   const {_} = useLingui()
+  const {showLoggedOut} = useLoggedOutView()
+  const {setShowLoggedOut} = useLoggedOutViewControls()
 
   return (
     <CenteredView style={[styles.container, pal.view]}>
+      {showLoggedOut && (
+        <Pressable
+          accessibilityRole="button"
+          style={{
+            position: 'absolute',
+            top: 20,
+            right: 20,
+            padding: 20,
+            zIndex: 100,
+          }}
+          onPress={() => setShowLoggedOut(false)}>
+          <FontAwesomeIcon
+            icon="x"
+            size={24}
+            style={{
+              color: 'white',
+            }}
+          />
+        </Pressable>
+      )}
```

Comment:
Adds a way to return to the previous screen if you arrived at this screen by attempting an "authed" action. Same on mobile.

<img width="1334" alt="Screenshot 2023-11-20 at 3 07 32 PM" src="https://github.com/bluesky-social/social-app/assets/4732330/33c7b175-35ff-4b8d-ab72-e42578a97e60">